pkg/controller/certificates/util.go

```diff
@@ -34,6 +34,12 @@ import (
 	"github.com/jetstack/cert-manager/pkg/util/pki"
 )
 
+// The amount of time after the LastFailureTime of a Certificate
+// before the request should be retried.
+// In future this should be replaced with a more dynamic exponential
+// back-off algorithm.
+const RetryAfterLastFailure = time.Hour
```

Comment:
We should look at having the Certificate duration being a variable to the backoff function. A failed 1 hour cert will always have downtime here.